packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -185,12 +182,24 @@ const HorizontalScroll = React.forwardRef<HTMLDivElement, Props>(
         >
           <div
             className={cx("relative inline-flex size-full", isDragging && "pointer-events-none")}
+            // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+            tabIndex={0}
```

Comment:
As discussed offline, pointing out the issues with outline of the outer div & not focusable arrows when children are focusable. But I don't remember what was the conclusion here, if the arrows should be focusable or not 🤔

Reply:
I believe arrows should be focusable, as they're button elements in the end, they'd be just another element to scroll.

Reply:
About the outline issue, check [this](https://github.com/kiwicom/orbit/pull/4599/files#r1932526246), you might be able to help :)